tests/performance/test_data_analysis/data_analyzer_module/test_data_analyzer_module.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (C) 2015, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+"""
+Statistical Data Analyzer Module Test
+
+
+This module contains a basic test that allows you to perform statistical analysis and calculations
+on two data sets and to make comparisons between them. This allows to detect significant
+differences between the two sets automatically.
+
+This test uses t-Student, Levene and ANOVA tests to detect possible significant differences in the
+metrics of the data sets. If such differences exist, comparisons are made between the main
+statistics with respect to a threshold value which, if exceeded, is marked as an error and
+reported conveniently.
+
+Tests:
+    - test_comparison: detects significant differences between metrics of two data sets.
+
+Issue: https://github.com/wazuh/wazuh/issues/24688
+"""
+
+import pytest
+
+from collections.abc import Callable
+from typing import Tuple
+from statistical_data_analyzer import DataLoader, StatisticalComparator, StatisticalTests
```

Comment:
```suggestion
from collections.abc import Callable
from typing import Tuple

import pytest

from statistical_data_analyzer import DataLoader, StatisticalComparator, StatisticalTests
```

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5622/commits/a8be0593badbbd04bb3211468f64ad2689d17bf3